File: src/main/java/no/fintlabs/linkwalker/auth/model/AuthObject.java
package no.fintlabs.linkwalker.auth.model;

import java.util.List;

public record AuthObject(
        String dn,
        String name,
        String shortDescription,
        String assetId,
        String asset,
        String note,
        String password,
        String clientSecret,
        String publicKey,
        String clientId,
        List<String> components,
        List<String> accessPackages,
        boolean managed
) {
}

File: src/main/kotlin/no/fintlabs/linkwalker/auth/model/AuthObject.kt
package no.fintlabs.linkwalker.auth.model

data class AuthObject(
    val dn: String,
    val name: String,
    val shortDescription: String,
    val assetId: String,
    val asset: String,
    val note: String,
    val password: String,
    val clientSecret: String,
    val publicKey: String,
    val clientId: String,
    val components: MutableList<String>,
    val accessPackages: MutableList<String>,
    val managed: Boolean
)
